docs/references/support_matrix.rst

```diff
@@ -49,3 +49,12 @@ Here is a list of platforms and releases that Ubuntu Pro Client, through the
       - amd64, arm64, armhf, ppc64el, riscv64, s390x
       - TBD
```

Comment:
Initial release on Mantic is `29.4`

Reply:
I can't highlight because it falls off the changes of the PR, but
Final release on Lunar is `30`
Final release on Kinetic is `28.1`